### DISCUSSION TITLE
I would like to filter the dependency dashboard by ecosystem (Python vs. JS vs. Docker)

### DISCUSSION THREAD

**Discussion by cclauss** [renovatebot/renovate#23346] [Suggest an Idea]
### Tell us more.

We have a busy dependency dashboard but...
* [ ] I do not know **Rust** so I would like to hide those dependencies.
* [ ] On Monday, I want to focus only on updating **Python** dependencies.
* [ ] On Wednesday, I want to focus only on updating **JavaScript** dependencies.
* [ ] Before my deployment, I want to ensure that all my **Kubernetes**, **Docker**, and **Helm** dependencies are up-to-date.

It would be great if there was a way to **filter** our dependency dashboard so that I could focus on the ecosystem of concern.  Being able to temporarily hide and show dependencies by ecosystem would allow me to focus on the job at hand without missing anything.

**Comment by HonkingGoose** [comment]
Your Dependency Dashboard has a section called `Detected dependencies`. It lists the dependencies from each ecosystem. It's not what you want, but it could help you for now.

We recently added categories to managers and implemented a matcher for package rules. I don't know if that would help us change group/filter the Dependency Dashboard? See:

- #16534

Your idea sounds similar to this issue that we closed as "won't do":

- #5441

I'll let the maintainers decide what to do with your ideas.

---

**Edit:** I found this issue which would allow you to filter based on the issue labels:

- https://github.com/renovatebot/renovate/issues/12414
